admin/sql/create_tables.sql

```diff
@@ -276,6 +276,15 @@ ALTER TABLE user_setting
     ADD CONSTRAINT user_setting_timezone_name_check
     CHECK ( timezone_name IS NULL OR now() AT TIME ZONE timezone_name IS NOT NULL );
 
+CREATE TABLE recommendation.do_not_recommend (
+    id              INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL,
+    user_id         INTEGER NOT NULL,
+    entity          do_not_recommend_entity_type NOT NULL,
+    entity_mbid     UUID NOT NULL,
+    until           TIMESTAMP WITH TIME ZONE,
```

Comment:
should this be DEFAULT NULL?

Reply:
Yup leaving out a default is implicitly default null.